style.css

```diff
@@ -130,3 +130,465 @@ p {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.content-data > p {
+  width: 327px;
+  height: 147px;
+  margin-top: 10px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.content-data > ul {
+  display: flex;
+  justify-content: space-between;
+  gap: 12px;
+  list-style: none;
+  padding-inline-start: inherit;
+}
+
+.list2 {
+  border: 1px solid white;
+  padding: 12px;
+}
+
+.list1 > a {
+  text-decoration: none;
+  padding: 12px;
+  text-align-last: center;
+  border: 1px solid black;
+  height: 40px;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #091e42;
+}
+
+.work1-btn {
+  list-style: none;
+}
+
+.work1-button {
+  padding: 12px;
+  width: 161px;
+  background-color: rgb(255, 107, 0);
+  border: none;
+  color: white;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  margin-top: 10px;
+}
+
+.work1-btn :hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.work-display {
+  display: grid;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.work-2 {
+  width: 327px;
+  margin: auto;
+  margin-top: 24px;
+  background: url(./svg/ImgPlaceholder1.svg);
+  background-color: #1c1a19;
+  background-repeat: no-repeat;
+  background-position-x: center;
+}
+
+.work2-content > * {
+  padding: 0 12px 0 12px;
+}
+
+.work2-content > h2 {
+  margin-top: 80px;
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  color: white;
+}
+
+.work2-content > h1 {
+  display: none;
+}
+
+.work2-content > p {
+  font-family: "Inter", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.work2-content ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 12px;
+  width: 247px;
+  height: 40px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.work2-content li {
+  display: flex;
+  flex-direction: row;
+  align-items: right;
+  padding: 8px 10px;
+  width: 67px;
+  height: 30px;
+  background: rgba(255, 255, 255, 0.24);
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border: none;
+}
+
+.work2-content ul li {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px;
+  width: 60px;
+  height: 25px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
```

Comment:
Great job:+1: on your progress so far! However, there are some further improvements needed to meet the previous reviewer's requirements:
- [ ] First, please make sure to write "HTML" in uppercase for consistency and adherence to standard naming conventions.:+1:
![image](https://github.com/sulaimanahmadi/Portfolio/assets/55840999/26956491-75d2-4fd9-8202-fb205b84f702)
- [ ] In each `<li>` element, I noticed that you used Flexbox. To perfectly align the items horizontally as required, you need to add justify-content: center; to your CSS code, please ensure it matches the design accurately.:+1:

| Your (without justify-center)       | Professional  (with justify-center) |
|---------------------|-----------------------|
|  ![image](https://github.com/sulaimanahmadi/Portfolio/assets/55840999/b5cc2ff4-1c12-4720-bdc9-422c477b9857) |  ![image](https://github.com/sulaimanahmadi/Portfolio/assets/55840999/34ccceea-d992-4a08-815f-75f6abb88c29) |

- [ ] Currently, you have `padding: 8` applied on all sides. However, it would be more appropriate to have a padding of 12 for the left and right sides and 10 for the top and bottom to meet the requirement. To understand how I arrived at these measurements, you can refer to [this document](https://help.figma.com/hc/en-us/articles/360039956974-Measure-distances-between-layers).:+1:

![image](https://github.com/sulaimanahmadi/Portfolio/assets/55840999/b1cc39f5-6388-4e2b-a4ce-b33db99bced7)

- [ ] Please remove the width: 60px; from your CSS code as it may restrict the size of the skills you display. Since some skills might be longer or wider than that specified width, it's better to allow them to adjust dynamically according to their content. Removing this constraint will ensure that your skills are displayed appropriately and avoid any potential visual inconsistencies.:+1: